server/routes/server.routes.js

```diff
@@ -1,48 +1,13 @@
 import { Router } from 'express';
 const router = new Router();
-import path from 'path';
-import { renderIndex } from '../views/index'
+import { renderIndex } from '../views/index';
 
 // this is intended to be a temporary file
 // until i figure out isomorphic rendering
 
-router.route('/').get((req, res) => {
-  res.send(renderIndex());
-});
-
-router.route('/signup').get((req, res) => {
-  res.send(renderIndex());
-});
-
-router.route('/projects/:project_id').get((req, res) => {
-  res.send(renderIndex());
-});
-
-// router.route('/full/:project_id').get((req, res) => {
-//   res.send(renderIndex());
-// });
-
-router.route('/login').get((req, res) => {
-  res.send(renderIndex());
-});
-
-router.route('/reset-password').get((req, res) => {
-  res.send(renderIndex());
-});
-
-router.route('/reset-password/:reset_password_token').get((req, res) => {
-  res.send(renderIndex());
-});
-
-router.route('/sketches').get((req, res) => {
-  res.send(renderIndex());
-});
-
-router.route('/about').get((req, res) => {
-  res.send(renderIndex());
-});
-
-router.route('/:username/sketches').get((req, res) => {
+// Wildcard route.
+// 404s are handled by the client (react-router) and redirect to the index route.
+router.route('*').get((req, res) => {
```

Comment:
Make sure that this gets imported last of all the other routers in`server/server.js`! Otherwise I think it will override some custom routes.

Also this wildcard route makes me nervous, though it maybe be totally unfounded. I guess I just prefer to whitelist things and understand what's happening rather than allow all options and blacklist from there. Again, could totally not make sense. 

But this also begs a UX question: is it better to redirect to the root url or render a 404 page? If the latter, I think the 404 page should be rendered server side. I'm open to both options.